velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -202,6 +203,8 @@ VeloxToSubstraitExprConvertor::toSubstraitLiteral(
           mapTypeKindToName(variantValue.kind()));
   }
 
+  literalExpr->set_nullable(true);
```

Comment:
Why do we set nullability for a literal unconditionally? Should it be set depending on whether the value is null or not, e.g. base on variantValue.isNull()?

Reply:
fix.